usaspending_api/etl/elasticsearch_loader_helpers/delete_data.py

```diff
@@ -278,7 +278,9 @@ def _gather_deleted_ids(config: dict) -> list:
         elif "afa_generated_unique" in data:
             new_ids = ["ASST_TX_" + x.upper() for x in data["afa_generated_unique"].values]
         else:
-            logger.info(format_log(f"[Missing valid col] in {obj.key}", action="Delete"))
+            msg = f"[Missing valid CSV col] in {obj.key}"
```

Comment:
Was this tested? There might be files in S3 which don't contain those names in the headers.

Reply:
I think there was only 3 types of files in there
- fabs
- fpds
- idv

The FPDS and IDV share the same two cols (one of which is `detached_award_proc_unique`). And FABS has `afa_generated_unique`. 

So we're covered. But if we add new files there, which imply things need to be deleted from ES, we want to know if we're silently skipping them by instead throwing an error.